### PR TITLE
Disabling .Net 7 target

### PR DESCRIPTION
**Pull request by trwalke**
Fixes #
Disabling .Net 7 target to fix build

**Changes proposed in this request**
Disabling .Net 7 target to fix build

**Testing**

**Performance impact**

**Documentation**
- [ ] All relevant documentation is updated.

### CODE REVIEW

**Review by pmaytak — APPROVED** [review]
Does this also need to be reverted?
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/2f83c8e783048734cdec26c4358c96d8f30a0b78/build/template-run-unit-tests.yaml#L46-L56

**Comment by trwalke** [comment]
> Does this also need to be reverted?
> 
> https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/2f83c8e783048734cdec26c4358c96d8f30a0b78/build/template-run-unit-tests.yaml#L46-L56

Yes, Thanks @pmaytak

File: tests/Microsoft.Identity.Test.Unit/Microsoft.Identity.Test.Unit.csproj [hunk 1]
```diff
@@ -16,9 +16,9 @@
     <Platforms>AnyCPU;x64</Platforms>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNet7)' or '$(TargetFramework)' == '$(TargetFrameworkNet7Win)'">
+  <!-- <PropertyGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNet7)' or '$(TargetFramework)' == '$(TargetFrameworkNet7Win)'">
```
Comment (bgavrilMS) [inline tests/Microsoft.Identity.Test.Unit/Microsoft.Identity.Test.Unit.csproj:19]: delete dead code.

**Review by bgavrilMS — APPROVED** [review]

File: tests/Microsoft.Identity.Test.Common/TestConstants.cs [hunk 2]
```diff
@@ -31,7 +31,7 @@ public static HashSet<string> s_scope
         public static readonly string[] s_graphScopes = new[] { "user.read" };
         public const uint JwtToAadLifetimeInSeconds = 60 * 10; // Ten minutes
         public const string ClientCredentialAudience = "https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47/v2.0";
-        public const string AutomationTestThumbprint = "4E87313FD450985A10BC0F14A292859F2DCD6CD3";
+        public const string AutomationTestThumbprint = "444B697D869032F29F9A162D711AF3E2791AD748";
```
Comment (bgavrilMS) [inline tests/Microsoft.Identity.Test.Common/TestConstants.cs:34]: 2 in 1 fix ;)

**Review by neha-bhargava — APPROVED** [review]

**Pull request merged**